docs/builtin-gadgets/trace/dns.md

```diff
@@ -83,11 +83,11 @@ The tool will list the different DNS queries that were made:
 $ sudo ig trace dns -c test-trace-dns
 RUNTIME.CONTAINERNAME                   PID        TID        COMM             QR TYPE      QTYPE      NAME                                   RCODE                NUMANSW…
 test-trace-dns                          38797      38798      isc-net-0000     Q  OUTGOING  A          inspektor-gadget.io.                                        0
-test-trace-dns                          38797      38798      isc-net-0000     R  HOST      A          inspektor-gadget.io.                   NoError              2
+test-trace-dns                          38797      38798      isc-net-0000     R  HOST      A          inspektor-gadget.io.                   No Error             2
 test-trace-dns                          38801      38802      isc-net-0000     Q  OUTGOING  AAAA       inspektor-gadget.io.                                        0
-test-trace-dns                          38801      38802      isc-net-0000     R  HOST      AAAA       inspektor-gadget.io.                   NoError              2
+test-trace-dns                          38801      38802      isc-net-0000     R  HOST      AAAA       inspektor-gadget.io.                   No Error             2
 test-trace-dns                          38807      38808      isc-net-0000     Q  OUTGOING  MX         inspektor-gadget.io.                                        0
-test-trace-dns                          38807      38808      isc-net-0000     R  HOST      MX         inspektor-gadget.io.                   NoError              3
+test-trace-dns                          38807      38808      isc-net-0000     R  HOST      MX         inspektor-gadget.io.                   No Error             3
 ```
 
 ### Limitations
```

Comment:
Could you please update the limitations?

- The number of answers is no anymore limited to 8 and, afaiu, the gopacket pkg support uncompressed messages.
- The remaining limitations are the lack of support for DNS over TCP and IPv6. In addition, we won't parse the packet correctly if the IPv4 header has the Options field set.